projects.go

```diff
@@ -88,6 +88,16 @@ type Project struct {
 	CIConfigPath *string            `json:"ci_config_path"`
 }
 
+// Project users represents a GitLab project.
```

Comment:
The comment should start with the same name as the type, so `ProjectUsers` in this case